js/codap-handler.js

```diff
@@ -137,8 +137,8 @@ export function generateCodapData (options) {
     if (output.showInCodap) {
       const rawValue = outputs[outputName]
       const outputValue = output.dispFunc
-                            ? output.dispFunc(rawValue)
-                            : rawValue
+        ? output.dispFunc(rawValue)
+        : rawValue
```

Comment:
Aside: I'm not a fan of this white-space convention. I think it sacrifices readability for consistency, but 🤷‍♂ .